firebase/functions/src/index.ts

```diff
@@ -324,3 +324,28 @@ export const contactMail =
           },
         });
       });
+
+export const subscribeToNewsletter = functions
+    .region(region)
+    .https.onCall(async (data: any, context: functions.https.CallableContext) => {
+      functions.logger.debug("subscribeToNewsletter parameters", data);
+
+      const {name, email} = data;
+
+      await db.collection("interested_users").add({
+        name,
+        email,
+      });
+
+      await db.collection("mail").add({
+        to: email,
+        message: {
+          subject: "Thank you for subscribing to Clothing Loop",
+          html: ` <p>Hi ${name},</p>
+                      <p>Thank you for subscribing!</p>
+                      <p>Regards,</p>
+                      <p>The clothing-loop team!</p>
+              `,
+        },
+      });
+    });
```

Comment:
You might need to merge with the main branch, this was already merged: https://github.com/CollActionteam/clothing-loop/pull/132/files

Reply:
I'm a bit lost with this workflow - the changes you've approved in the PR #132, were merged into the branch used for this PR. I thought I'd need to create a PR with both changes (as they're both part of the same workflow, if it makes sense). What's the right approach in this case? Were the changes from PR #132 meant to be merged with main?

Reply:
@giuliamml @emer7 I see that that pull request was merged into "newsletter", not into main. I think that caused the confusion here.

There are a few rules I think we should adhere to:

* Code should be reviewed once, when it's being pushed into main. There's no point in reviewing the same code multiple times (like here). Reviewing costs time. 
* Pull requests should contain finished functionality that works, and be as small as possible. Pull requests that are too big make it more likely things slip through, so keep things small if possible. I'd have expected the newsletter PR to be merged in main since it works as-is. 

Reviewing at multiple levels might be ok with very big projects, but there's not much of a point here.

If you need to move code between branches, just use "git merge" instead of a pull request.

I'll review this for now, but can you guys stick to those rules in the future?

Reply:
agreed! thank you